app/feature/feature-help-center/src/main/kotlin/com/hedvig/android/feature/help/center/home/HelpCenterHomeDestination.kt

```diff
@@ -190,27 +186,47 @@ private fun HelpCenterHomeScreen(
                         is HelpCenterUiState.QuickLinkType.CommonClaimType -> onNavigateToCommonClaim(
                           quickLink.commonClaim,
                         )
+
                         is HelpCenterUiState.QuickLinkType.QuickActionType -> onQuickActionsSelected(
                           quickLink.quickAction,
                         )
                       }
                     },
-                    modifier = Modifier.fillMaxWidth(),
+                    modifier = Modifier
+                      .fillMaxWidth()
+                      .padding(horizontal = 16.dp),
                   ) {
                     Column(
                       verticalArrangement = Arrangement.Center,
-                      horizontalAlignment = Alignment.CenterHorizontally,
                     ) {
+                      Spacer(modifier = Modifier.height(12.dp))
                       Text(
                         text = when (quickLink) {
                           is HelpCenterUiState.QuickLinkType.CommonClaimType -> quickLink.commonClaim.title
                           is HelpCenterUiState.QuickLinkType.QuickActionType -> stringResource(
                             quickLink.quickAction.titleRes,
                           )
                         },
-                        textAlign = TextAlign.Center,
-                        modifier = Modifier.padding(16.dp),
+                        textAlign = TextAlign.Start,
+                        modifier = Modifier.padding(horizontal = 16.dp),
+                      )
+                      Spacer(modifier = Modifier.height(4.dp))
+                      Text(
+                        text = when (quickLink) {
+                          is HelpCenterUiState.QuickLinkType.CommonClaimType -> {
+                            val hintTextRes = quickLink.commonClaim.hintTextRes
+                            if (hintTextRes != null) stringResource(hintTextRes) else ""
```

Comment:
If we do in fact get an empty string here, do we want the text to still show a bit off-center as it will, or should it become centered vertically in the card, or should the card itself shrink?
I assume design does not handle this case right?

Reply:
Nope, but again - there are only two common claims now, and both have hint text, so I think that's why they didn't make the design. I thought that in an unlikely case we don't have the hint, it would be more logical to have the title a bit off-center, so all cards look alike

Reply:
Yeap sounds good to me